src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmServiceImpl.java

```diff
@@ -137,7 +137,7 @@ public boolean deleteBlobData(String containerName, String blobPathAndName) {
             }
 
         } catch (Exception e) {
-            log.error("Could not delete from container: " + containerName + " blobPathAndName: " + blobPathAndName, e.getMessage(), e);
+            log.error("Could not delete from container {} blobPathAndName {} - {}", containerName, blobPathAndName, e.getMessage(), e);
```

Comment:
Wondering why we need the exception message and stacktrace, does stacktrace not give everything anyway?

Reply:
I don't generally use stacktrace unless theres a real need. I was debating whether it should have stacktrace or not.